docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+   <td><span style="text-decoration:underline;">宮崎</span></td>
+   <td>YES</td>
+   <td>ja</td>
+   <td>surname First</td>
+   <td></td>
+  </tr>
+  <tr>
+   <td colspan="7" ><span style="text-decoration:underline;">宮崎</span>駿</td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+   <td colspan="7" ><strong>formatting locale: de_CH</strong>, formatting locale script is Latn</td>
+  </tr>
+  <tr>
+   <td><strong>name locale</strong></td>
+   <td><strong>given</strong></td>
+   <td><strong>surname</strong></td>
+   <td><strong>same<br>script</strong></td>
+   <td><strong>formatting<br>locale</strong></td>
+   <td><strong>order</strong></td>
+   <td><strong>foreign<br>space</strong></td>
+  </tr>
+  <tr>
+   <td>de_Latn_CH</td>
+   <td>Albert</td>
+   <td>Einstein</td>
+   <td>YES</td>
+   <td>de</td>
+   <td>given First</td>
+   <td></td>
+  </tr>
+  <tr>
+   <td colspan="7" >“Albert Einstein”</td>
+  </tr>
+  <tr>
+   <td>de_Jpan_CH</td>
+   <td>アルベルト</td>
+   <td>アインシュタイン</td>
+   <td>NO</td>
+   <td>ja<br>from script</td>
+   <td>given First</td>
+   <td>“`・`”</td>
+  </tr>
+  <tr>
+   <td colspan="7" >“アルベルト`・`アインシュタイン”</td>
+  </tr>
+  <tr>
+   <td>und_Latn_JP</td>
+   <td>Hayao</td>
+   <td>Miyazaki</td>
+   <td>YES</td>
+   <td>und</td>
+   <td>given First</td>
+   <td>“` `”</td>
+  </tr>
+  <tr>
+   <td colspan="7" >“Hayao Miyazaki”</td>
+  </tr>
+</table>
+
+## 7 <a name="7-sample-name" href="#7-sample-name">Sample Name</a>
+
+The sampleName element is used for test names in the personNames LDML data for each locale to aid in testing and display in the CLDR Survey Tool. They are not intended to be used in production softwareas prompts or placeholders in a user interface and should not be displayed in a user interface.
+
+### 7.1 <a name="7-1-syntax" href="#7-1-syntax">Syntax</a>
+
+```xml
+<!ELEMENT sampleName ( nameField+ )  >
+<!ATTLIST sampleName item NMTOKENS #REQUIRED >
+```
+
+* `NMTOKENS` must be one of `( givenOnly | givenSurnameOnly | given12Surname | full )`
+
+### 7.2 <a name="7-2-expected-values" href="#7-2-expected-values">Expected values</a>
+
+For the expected sample name items, assume a name such as Mr. Richard “Rich” Edward Smith Iglesias Ph.D.
+
+* `givenOnly` is for an example name with only the given is presented: “Richard” or “Rich” (informal)
+* `givenSurnameOnly` is for an example name with only the given name and surname: “Richard Smith” or “Rich Smith” (informal)
+* `given12Surname` is for an example using all given names and a surname: “Richard Edward Smith” and “Rich E. Smith” (informal)
+* `full` is used to present a name using all fields: “Mr. Richard Edward Smith Iglesias, Ph.D.”
+
+The `nameField` values and their modifiers are described in the [Person Name Object](#3-person-name-object) and [namePattern Syntax](#5-namepattern-syntax) sections.
+
+## 8 <a name="8-personname-data-interface-examples" href="#8-personname-data-interface-examples">PersonName Data Interface Examples</a>
+
+### Example 1. {#example-1}
```

Comment:
the {#example 1} needs to be changed to a link reference, same below.

Reply:
Thanks, fixed. These were still in the form created by the conversion from GoogleDocs. Fixed them to the standard form.